.github/workflows/cdk-test.yml

```diff
@@ -0,0 +1,36 @@
+name: cdk test
+
+defaults:
+  run:
+    working-directory: notificationworkerlambda/cdk
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "notificationworkerlambda/cdk/**"
+
+  push:
+    branches:
+      - master
+    paths:
+      - "notificationworkerlambda/cdk/**"
+
+jobs:
+  cdk-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16"
```

Comment:
Minor: it might be worth adding a `.nvmrc` file* and [providing the `node-version-file` input](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file). 

This would help to ensure that developers (when running this locally) and CI use the same version of node.

*Personally I would put this at the project root - do you think that makes sense in this context @JamieB-gu?

Reply:
> Minor: it might be worth adding a .nvmrc file* and [providing the node-version-file input](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file).

I hadn't realised they'd added this to the action! Yes, this sounds like a great idea.

> Personally I would put this at the project root - do you think that makes sense in this context

👍. I _think_ nvm will check parent directories for an `.nvmrc`, so even if you run `nvm use` from inside the `cdk` directory it _should_ still work? @frankie297 it might be worth testing this though.

Reply:
👍

Reply:
Added a .nvmrc file at the root of the project and tested